docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,490 @@
+possible classes. In this case, there are two classes: `true` and `false`. The 
+class names along with the probability of the given classes are displayed in the 
+`top_classes` object. The most probable class is the prediction. In the example 
+above, `false` has a `class_probability` of 0.93 while `true` has only 0.06, so 
+the prediction will be `false` which coincides with the ground truth contained 
+by the `FlightDelay` field. The class probability values help you understand how 
+sure the model is about the prediction. The higher number means that the model 
+is more confident.
+
+If a document doesn't contain a prediction field, then it is excluded from the 
+analysis. In order to be analyzed, a document must contain at least one field 
+with a supported data type (`numeric`, `boolean`, `text`, `keyword` or `ip`) and 
+must not contain arrays with more than one item.
+--
+
+
+[[flightdata-classification-evaluate]]
+==== Evaluating results
+
+The results can be evaluated for documents which contain both the ground truth 
+field and the prediction. In the example below, `FlightDelay` contains the 
+ground truth and the prediction is stored as `FlightDelay_prediction`.
+
+. Use the {dfanalytics} evaluate API to evaluate the results.
++
+--
+First, we want to know the training error that represents how well the model 
+performed on the training dataset:
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/_evaluate
+{
+ "index": "df-flight-delayed",  <1>
+   "query": {
+    "term": {
+      "ml.is_training": {
+        "value": true  <2>
+      }
+    }
+  },
+ "evaluation": {
+   "classification": {
+     "actual_field": "FlightDelay",  <3>
+     "predicted_field": "ml.FlightDelay_prediction",  <4>
+     "metrics": {  
+       "multiclass_confusion_matrix" : {}
+     }
+   }
+ }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> The destination index which is the output of the analysis job.
+<2> We calculate the training error by only evaluating the training data.
+<3> The field that contains the ground truth label.
+<4> The field that contains the predicted value.
+
+Next, we calculate the generalization error that represents how well the model 
+performed on previously unseen data:
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/_evaluate
+{
+ "index": "df-flight-delayed",
+   "query": {
+    "term": {
+      "ml.is_training": {
+        "value": false  <1>
+      }
+    }
+  },
+ "evaluation": {
+   "classification": {
+     "actual_field": "FlightDelay",
+     "predicted_field": "ml.FlightDelay_prediction",
+     "metrics": {  
+       "multiclass_confusion_matrix" : {}
+     }
+   }
+ }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> By only evaluating the data that was not used in training, we can 
+calculate the generalization error which shows the algorithm accuracy in making 
```

Comment:
I would use the word `model` instead of the word `algorithm` here, because `model` is the one usually associated with a particular accuracy. 

`which shows the model's accuracy in making 
predictions for previously unseen data.`